PYME/DSView/fitInfo.py

```diff
@@ -163,7 +163,15 @@ def UpdateDisp(self, index):
         
     def DrawOverlays(self, vp, dc):
         do = vp.do
-        frameResults = self.fitResults[self.fitResults['tIndex'] == do.zp]
```

Comment:
with the 5D model, we should really be checking for t first, will modify accordingly.